config.json

```diff
@@ -17,8 +17,12 @@
 		{"url": "http://example.com/en/sitemap.xml", "lang": "en"}
 	],
 	"http": {
-		"auth": ""
+		"auth": "",
+		"headers": {
+
+		}
 	},
+	"delayBetweenRequest": 100,
```

Comment:
Can you please add this in the docs (README.md)

Reply:
Sure.

Reply:
👍